tests/profile/test_jsonschema_profiler.py

```diff
@@ -279,6 +354,58 @@ def test_profile_enum_schema(empty_data_context, enum_types_schema):
                 "value_set": ["XS", "S", "M", "XL", "XXL"],
             },
         },
+        {
+            "expectation_type": "expect_column_values_to_not_be_null",
+            "kwargs": {"column": "shirt-size"},
+            "meta": {},
+        },
+        {
+            "meta": {},
+            "expectation_type": "expect_column_to_exist",
+            "kwargs": {"column": "optional-color"},
+        },
+        {
+            "meta": {},
+            "expectation_type": "expect_column_values_to_be_in_set",
+            "kwargs": {
+                "column": "optional-color",
+                "value_set": ["red", "green", "blue"],
+            },
+        },
+        {
+            "meta": {},
+            "expectation_type": "expect_column_to_exist",
+            "kwargs": {"column": "optional-hat"},
+        },
+        {
+            "kwargs": {
+                "column": "optional-hat",
+                "type_list": list(ProfilerTypeMapping.STRING_TYPE_NAMES),
+            },
+            "expectation_type": "expect_column_values_to_be_in_type_list",
+            "meta": {},
+        },
+        {
+            "meta": {},
+            "expectation_type": "expect_column_values_to_be_in_set",
+            "kwargs": {
+                "column": "optional-hat",
+                "value_set": ["red", "green", "blue"],
+            },
+        },
+        {
+            "meta": {},
+            "expectation_type": "expect_column_to_exist",
+            "kwargs": {"column": "optional-answer"},
+        },
+        {
+            "meta": {},
+            "expectation_type": "expect_column_values_to_be_in_set",
+            "kwargs": {
+                "column": "optional-answer",
+                "value_set": ["yes", "no", "null"],
```

Comment:
I'm thinking out loud a bit here. Should this value set be `["yes", "no", None]`? Even though GE ignores null values by default? Or perhaps it would be more accurate if the set were `["yes", "no"]`?

Reply:
Thanks for the catch here. We do have a bug as this is a dict. Below is how python deserializes a json with null into an object.

```python
import json
data = json.loads('{"key":null}')
print(data['key'])
# output: None
```

Fixed it in the latest commit

Reply:
Wonderful!